packages/react/src/components/Accordion/AccordionHeading.tsx

```diff
@@ -1,73 +1,32 @@
-import { ChevronDownIcon } from '@navikt/aksel-icons';
 import cl from 'clsx/lite';
-import type { HTMLAttributes, MouseEventHandler, ReactNode } from 'react';
-import { forwardRef, useContext } from 'react';
+import type { HTMLAttributes, ReactNode } from 'react';
+import { forwardRef } from 'react';
 
-import { Heading, Paragraph } from '../Typography';
-
-import { AccordionItemContext } from './AccordionItem';
+import { Paragraph } from '..';
```

Comment:
This needs to point to the actual component to avoid circular dependencies 
```suggestion
import { Paragraph } from '../Typography';
```

<img width="1248" alt="image" src="https://github.com/user-attachments/assets/5f7a762a-0ff7-4f8c-808b-2bc45349d500">